homeassistant/components/alexa/resources.py

```diff
@@ -260,6 +262,10 @@ def __init__(self, labels: list[str], ordered: bool = False) -> None:
         self._mode_ordered: bool = ordered
 
     def add_mode(self, value: str, labels: list[str]) -> None:
+        """Add mode to the supportedModes object."""
+        self.add_mode_with_locale(value, {label: "" for label in labels})
+
+    def add_mode_with_locale(self, value: str, labels: dict[str, str]) -> None:
         """Add mode to the supportedModes object."""
         self._supported_modes.append({"value": value, "labels": labels})
```

Comment:
We do we implement 2 methods here as we do not seem to use the empty label.

Reply:
Sorry i think i don't fully understand you.
So we have 2 methods here, one to add just a label and one to add the label with a locale. Because the _supported_modes attribute does only accept a label with a locale a placeholder is set as locale.
Later when the _supported_modes field is added to the capabilities object the placeholder is replaced by the default locale en-US. See line 244 in resources.py.
I did this because then there is only one place that uses the default locale. If you think that is confusing i can just replace the placeholder with the default locale.